app/template/nav.php

```diff
@@ -2,19 +2,13 @@
 $menu = MainMenu::create()
     ->addItem("Événements", "/evenements", "fa-calendar")
     ->addItem("Les licenciés", "/licencies", "fa-users")
-    ->addItem("Mon profil", "/mon-profil", "fa-gear")
     ->addItem("Suggestions", "/feedback/nouveau", "fa-lightbulb")
     ->addItem("Documents", "/documents", "fa-file")
     ->addItem("Liens utiles", "/liens-utiles", "fa-arrow-up-right-from-square");
 
 if (dev_or_staging()) {
     $menu->addItem("Dev", "/dev", "fa-code");
 }
-if (Feature::Messages->on()) {
-    $menu->addItem("Messages", "/messages", "fa-message");
-}
-
-$main_user = User::getMain();
```

Comment:
in my opinion we want to keep the messages on the side nav, as it is not yet a functionnality that is used a lot

Reply:
I took inspiration from instagram.

I say we add new functionality in the final way than we want it to be. That's why we have features!

Reply:
Ok !